engines/e_afalg.c

```diff
@@ -125,7 +125,17 @@ static ossl_inline int io_getevents(aio_context_t ctx, long min, long max,
                                struct io_event *events,
                                struct timespec *timeout)
 {
+#if defined(__NR_io_getevents)
     return syscall(__NR_io_getevents, ctx, min, max, events, timeout);
+#elif __TIMESIZE == 64
```

Comment:
Is `__TIMESIZE` a glibc specific thing or part of a wider standard?
Not all Linux platforms use glibc.

I also wonder if checking for `__NR_io_pgetevents_time64` being defined might be a better first option.

Reply:
Yes, it appears to only be a glibc thing. I guess a runtime check on sizeof(time_t) is a better option here.

I can check on __NR_io_pgetevents_time64 being defined, but there shouldn't be a way that neither is defined

Reply:
I have fixed this in the next version.